rest_framework/serializers.py

```diff
@@ -539,14 +539,6 @@ def __getitem__(self, key):
             return JSONBoundField(field, value, error)
         return BoundField(field, value, error)
 
-    # Include a backlink to the serializer class on return objects.
-    # Allows renderers such as HTMLFormRenderer to get the full field info.
-
-    @property
-    def data(self):
-        ret = super().data
-        return ReturnDict(ret, serializer=self)
```

Comment:
This is breaking change. You should not delete this property many projects depend on it